### DISCUSSION TITLE
Curated list of exercises in local wger instance (not synced w/ wger.de)

### DISCUSSION THREAD

**Discussion by seyfeb** [wger-project/wger#1731] [Q&A] [answered]
The list of exercises (and ingredients) on the wger.de instance is incredibly long, of varying quality, and a lot of duplicates exist, sometimes without even English translation. Therefore, on my own instance, I'd like to create a curated list of exercises so I do not have to search through the list of existing exercises. I might transfer some of the existing ones, but I could recreate them on my instance.

What would be the correct way to delete all existing exercises and disable synchronization with wger.de?

Context: I'm using the docker image and have set the environment variables

```
SYNC_EXERCISES_ON_STARTUP=False
DOWNLOAD_EXERCISE_IMAGES_ON_STARTUP=False
SYNC_INGREDIENTS_CELERY=False
SYNC_EXERCISES_CELERY=False
SYNC_EXERCISE_IMAGES_CELERY=False
SYNC_EXERCISE_VIDEOS_CELERY=False
```

in `config/wger-local.env` but after the containers have started up there are already 20 pages of exercises available in the instance.

**Comment by rolandgeider** [comment, marked as answer]
Hi! You will need to empty the exercise table in the database, there are some local fixtures with data that always gets loaded. Just do

`docker compose exec db psql -U wger wger -c "DELETE FROM exercises_alias; DELETE FROM exercises_exercisecomment; DELETE FROM exercises_exercise; DELETE FROM exercises_exercisebase_equipment;DELETE FROM exercises_exercisebase;"` and add your own

**Reply by seyfeb** [reply]
Thanks! This nearly worked ;) There where some foreign-key constraints that needed to be resolved first. I'm not sure what the correct order of deletions is, because I deleted from multiple tables until no errors were printed anymore when running 

`docker compose exec db psql -U wger wger -c "DELETE FROM exercises_exercisebase;"`

At least, I also needed to run

`docker compose exec db psql -U wger wger -c "DELETE FROM exercises_exercisebase_muscles;"`

and

`docker compose exec db psql -U wger wger -c "DELETE FROM exercises_exercisebase_muscles_secondary;"`

Btw., I hope my question was not perceived as some kind of criticism. I really appreciate the work done collecting all these exercises and might use several of them - I was just getting a little overwhelmed!
